xml/System/IntPtr.xml

```diff
@@ -1017,8 +1021,8 @@ For <xref:System.Numerics.IFloatingPoint%601>, this method matches the IEEE 754:
         <Parameter Name="value2" Type="System.IntPtr" />
       </Parameters>
       <Docs>
-        <param name="value1">The first pointer or handle to compare.</param>
-        <param name="value2">The second pointer or handle to compare.</param>
+        <param name="value1">The first signed integer to compare.</param>
```

Comment:
APIs in the other integer types use `64-bit signed integer`, as an example. 

It's not as easy to describe this the same way without using `ptr-bit signed integer` so I just left it as `signed integer`